Python/Product/PythonTools/PythonTools/Environments/AddCondaEnvironmentControl.xaml

```diff
@@ -0,0 +1,405 @@
+                                    <TextBlock
+                                        Name="ErrorMessage"
+                                        Text="{Binding Path=CondaPreview.ErrorMessage}"
+                                        Visibility="Collapsed"/>
+                                    <TextBlock
+                                        Name="EnvFileMessage"
+                                        Text="{Binding Path=CondaPreview.EnvFileContents}"
+                                        Visibility="Collapsed"/>
+                                    <ItemsControl
+                                        Name="PackagesList"
+                                        ItemsSource="{Binding Path=CondaPreview.Packages}"
+                                        Grid.IsSharedSizeScope="True"
+                                        Focusable="False"
+                                        Visibility="Collapsed">
+
+                                        <ItemsControl.ItemTemplate>
+                                            <DataTemplate>
+                                                <Grid>
+                                                    <Grid.ColumnDefinitions>
+                                                        <ColumnDefinition SharedSizeGroup="A" Width="Auto"/>
+                                                        <ColumnDefinition Width="8"/>
+                                                        <ColumnDefinition SharedSizeGroup="B" Width="Auto"/>
+                                                    </Grid.ColumnDefinitions>
+                                                    <TextBlock
+                                                        Grid.Column="0"
+                                                        Margin="0 0 0 7"
+                                                        HorizontalAlignment="Left"
+                                                        Text="{Binding Name}"/>
+                                                    <TextBlock
+                                                        Grid.Column="2"
+                                                        Margin="0 0 0 7"
+                                                        HorizontalAlignment="Left"
+                                                        Text="{Binding Version}"/>
+                                                </Grid>
+                                            </DataTemplate>
+                                        </ItemsControl.ItemTemplate>
+                                    </ItemsControl>
+                                </StackPanel>
+                            </ScrollViewer>
+                        </Grid>
+                        <DataTemplate.Triggers>
+                            <DataTrigger Binding="{Binding Path=CondaPreview.IsInProgress}" Value="True">
+                                <Setter TargetName="ProgressBar" Property="UIElement.Visibility" Value="Visible"/>
```

Comment:
`Property="Visibility"` should work

Reply:
It did. The whole thing is gone now, switched to using converters. It's much nicer.